The detection of OPENSSL_VERSION_NUMBER is fragile and stopped working.
We know we have at least openssl 1.1.
--- setup.py.orig	2021-11-08 08:54:46.000000000 +1100
+++ setup.py	2021-11-08 09:04:30.000000000 +1100
@@ -874,7 +874,7 @@
             missing.append('_ssl')
 
         # find out which version of OpenSSL we have
-        openssl_ver = 0
+        openssl_ver = 0x101010cf
         openssl_ver_re = re.compile(
             '^\s*#\s*define\s+OPENSSL_VERSION_NUMBER\s+(0x[0-9a-fA-F]+)' )
 
